### PR TITLE
Add network zone matching to criteo adapter

### PR DESCRIPTION
**Pull request by Swiiip**
## Type of change
- [ x] Feature

## Description of change
Add network zone matching feature for Criteo Adapter

- contact email of the adapter’s maintainer: i.caroulle@criteo.com h.duthil@criteo.com

### CODE REVIEW

File: modules/criteoBidAdapter.js [hunk 1]
```diff
@@ -35,22 +35,33 @@ var CriteoAdapter = function CriteoAdapter() {
...
-            bid.transactionId
+            bid.transactionId,
+            sizes.map((size) => {
+              return { width: size[0], height: size[1] }
+              }
+            )
           )
         );
 
+        if (!networkid && bid.params.networkId != undefined) {
```
Comment (dbemiller) [inline modules/criteoBidAdapter.js]: Is there any way for `networkid` to be defined here? I don't see any assignments between here and its declaration.

Comment (mkendall07) [inline modules/criteoBidAdapter.js]: agree, doesn't need to be there.

Comment (Swiiip) [inline modules/criteoBidAdapter.js]: Updated

**Comment by muuki88** [comment]
@Swiiip thanks for this addition. We are currently testing the sizes parameter and are wondering how the criteo ad server must be configured to handle multiple sizes per zoneId.

At the moment we send a bid request per zoneId/size combination, which leads to multiple bid responses per ad unit. Prebid is not designed to handle this and chooses the last bid response, not taking the cpm into account.

**Pull request merged**